apps/web/lib/auth.ts

```diff
@@ -0,0 +1,41 @@
+import GithubProvider from "next-auth/providers/github";
+import GoogleProvider from "next-auth/providers/google";
+import { PrismaAdapter } from "@auth/prisma-adapter";
+import { PrismaClient } from "@prisma/client";
+import type { Adapter } from "next-auth/adapters";
+import { SessionStrategy } from "next-auth";
+
+const prisma = new PrismaClient();
+
+export const authOptions = {
+  adapter: PrismaAdapter(prisma) as Adapter,
+  providers: [
+    GithubProvider({
+      clientId: process.env.GITHUB_ID || "",
+      clientSecret: process.env.GITHUB_SECRET || "",
+      allowDangerousEmailAccountLinking: true,
+    }),
+    GoogleProvider({
+      clientId: process.env.GOOGLE_CLIENT_ID || "",
+      clientSecret: process.env.GOOGLE_CLIENT_SECRET || "",
+      allowDangerousEmailAccountLinking: true,
```

Comment:
It was giving an error [OAuthAccountNotLinked](https://github.com/nextauthjs/next-auth/issues/519#issuecomment-667551224), when logging in from google, logging out and again logging with github, this is the solution I find to avoid the error. Let me know if it's a known error to you @hkirat and if there is any other way to solve it.

Reply:
its because - the gmail account is the same , so it gives error

Reply:
Yeah, correct

Reply:
can ignore for now